### PR TITLE
Migrate System Info Page to B3

### PR DESCRIPTION
**Pull request by biyeun**
@sravfeyn 
cc @benrudolph 
screen shot 2016-01-04 at 7 26 10 pm

### CODE REVIEW

**Comment by benrudolph** [comment]
big :bee: on a tear

File: corehq/apps/hqadmin/templates/hqadmin/system_info.html [hunk 1]
```diff
@@ -351,303 +349,327 @@
...
-                                {{ v.message }}
-                            </dd>
-                        {% endfor %}
-                    {% endfor %}
-                    </dl>
-                </td>
-            </tr>
-            </tbody>
-        </table>
+    <div class="row" style="margin-top: 15px;">
```
Comment (benrudolph) [inline corehq/apps/hqadmin/templates/hqadmin/system_info.html]: inline style seems odd

**Comment by benrudolph** [comment]
nothing blocking from my side. gonna go ahead and merge

**Comment by czue** [comment]
:fire:

**Pull request merged**